docs/reference/transform/setup.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[transform-setup]]
+=== Set up {transforms}
+++++
+<titleabbrev>Setup</titleabbrev>
+++++
+
+To use the {transforms}, you must have the
+{subscriptions}[appropriate license] and at least one
+<<transform-setup-nodes,{transform} node>> in your {es} cluster. If {stack}
+{security-features} are enabled, you must also ensure your users have the
+<<transform-privileges,necessary privileges>>.
+
+[discrete]
+[[transform-setup-nodes]]
+==== {transform-cap} nodes
+
+To use {transforms}, there must be at least one node in your cluster with
+`xpack.transform.enabled` and `node.transform` set to `true`. All data nodes and
+master-eligible nodes must have `xpack.transform.enabled` set to `true`, which
+is the default behavior.
```

Comment:
Maybe an additional sentence (or two): 

That means, you do not need to do anything, transforms works out of the box. However in special cases you might want to control where the transform task is running.